docs/guides/top/ebpf.md

```diff
@@ -40,17 +40,18 @@ Some eBPF programs of type `Kprobe` should pop up, including their runtime and r
 
 ```bash
 $ kubectl gadget top ebpf
-NODE             PROGID   TYPE             NAME             PID     COMM                      RUNTIME   RUNCOUNT
-minikube         6346     Kprobe           vfs_write_entry  540621  gadgettracerman        3.948619ms        378
-minikube         6345     Kprobe           vfs_read_entry   540621  gadgettracerman         677.012µs       1157
-minikube         6347     Tracing          gadget_ebpftop   540621  gadgettracerman          65.069µs       1101
-minikube         26       CGroupDevice                                                        3.667µs          2
-minikube         62       CGroupDevice                      2378    systemd                        0s          0
-minikube         53       CGroupDevice                                                             0s          0
+NODE             PROGID   TYPE             NAME             PID     COMM                      RUNTIME   RUNCOUNT      MAPMEMORY MAPCOUNT
+minikube         3459     Kprobe           vfs_read_entry   4169649 gadgettracerman          736.04µs       1288     40.953 MiB        4
+minikube         3458     Tracing          gadget_ebpftop   4169649 gadgettracerman          69.607µs       1088         12   B        1
+minikube         3460     Kprobe           vfs_write_entry  4169649 gadgettracerman         614.455µs        647     40.953 MiB        4
+minikube         24       CGroupDevice                                                        2.458µs          2          0   B        0
+minikube         60       CGroupDevice                      1765    systemd                        0s          0          0   B        0
+minikube         51       CGroupDevice                                                             0s          0          0   B        0
 ...
 ```
 
-So in this case for example, in the past second `vfs_write_entry` has been called 378 times, which took 3.948619ms.
+So in this case for example, in the past second `vfs_write_entry` has been called 647 times, which took 614.455µs.
+The program references 4 maps that have a total maximum size of 40.953 MB.
```

Comment:
I think the documentation should make it clearer that the size is not the current memory allocation but the maximum, calculated with `max_entries * (key_size + value_size)`, with those 3 parameters as visible from userspace. Some of the consequences:
* BPF_MAP_TYPE_PERF_EVENT_ARRAY: value_size is not counting the ring buffers, but only their file descriptors (i.e. sizeof(int) = 4 bytes)
* BPF_MAP_TYPE_{HASH,ARRAY}_OF_MAPS: value_size is not counting the inner maps, but only their file descriptors (i.e. sizeof(int) = 4 bytes)
* PERCPU maps: the reported size is for one cpu, the multiplication by the number of cpus is left as an exercise to the reader

Reply:
Added 👍